packages/toolpad-app/src/toolpad/Home.tsx

```diff
@@ -469,6 +475,72 @@ function AppsListView({ loading, apps, activeDeploymentsByApp, setDeletedApp }:
   );
 }
 
+function UpdateBanner() {
```

Comment:
personally i would like to have this in a separate component, i was working on the Home file the other day and it's a bit long to navigate - i was kind of expecting only the main view with the list of apps to be in this file

Reply:
+1 on turning /Home.tsx into a folder /Home/index.tsx and split the file

Reply:
I've started the effort in this PR and created https://github.com/mui/mui-toolpad/issues/1037 to track further efforts

Reply:
@bharatkashyap looks good to me, thanks for moving the component! feel free to merge